app/bingey-ui/src/components/Watchlist/TitleCard/TitleCard.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import './cardStyle.scss';
+//import TitleDialog from './TitleDialog';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
+    margin: 25,
+    height: 400,
+  },
+});
```

Comment:
As we decided to use Sass for styling and this doesn't use specific Mui hook's functionality, I think this should go to the `.scss` file